pyaerocom/variable.py

```diff
@@ -32,36 +32,36 @@ def __init__(self, var_name):
         self._nums = []
         try:
             self._nums = self._numbers_in_string(var_name)
-        except Exception:
+        except:
```

Comment:
```suggestion
        except Exception:
```

Reply:
@ejgal it is always good to catch at least the base Exception (otherwise code won't stop even on `CTRL+C`)

Reply:
Not sure what happened here, I was not supposed to edit that function at all.